src/Cache/Engine/XcacheEngine.php

```diff
@@ -77,10 +77,14 @@ public function write($key, $value)
     {
         $key = $this->_key($key);
 
+        if (!is_numeric($value)) {
```

Comment:
@jippi It would be good optimization. As I focused a bug fix and followed an implementation of RedisEngine, I didn't much care about other data types except for integer and float.